server/server.js

```diff
@@ -115,6 +116,33 @@ app.get(`${apiBaseUrl}/milestone-report`,
     return res.json({ data: result });
   }));
 
+// Get hired report
+app.get(`${apiBaseUrl}/milestone-report/csv/hired`,
+  keycloak.allowRolesMiddleware('ministry_of_health'),
```

Comment:
I noticed this was missing getUserInfoMiddleware and wondered why - then I realized we need to add logging lol

I think just logging the user id + action is sufficient here, we shouldn't need to capture any other details

Reply:
Totally missed that 🤦🏻 thanks!!